definitions/remote-admin.json

```diff
@@ -1,8 +1,61 @@
 {
+    "AweRay (AweSun)": {
+        "process_name": ["aweray_remote*.exe"],
+        "domain": ["asapi.aweray.net", 
+                   "asapi-us.aweray.net"],
+        "digsig_publisher": ["AWERAY PTE. LTD."]
+    },
     "Ammyy Admin": { 
         "process_name": ["aa_v*.exe"],
         "domain": ["ammyy.com"]
     },
+    "AeroAdmin" : {
+        "process_name": ["AeroAdmin.exe"],
+        "domain": ["aeroadmin.com"],
+        "digsig_publisher": ["AeroAdmin LLC"]
+    },
+    "AnyDesk" :{
+        "process_name": ["anydesk.exe"],
+        "domain": ["net.anydesk.com"],
+        "digsig_publisher": ["philandro Software GmbH"]
+    },
+    "AnyViewer" : {
+        "process_name": ["AnyViewerSetup.exe", 
+                        "RCClient.exe", 
+                        "RCService.exe"],
+        "domain": ["aomeisoftware.com"],
+        "digsig_publisher": ["AOMEI International Network Limited"]
+    },
+    "Atera": {
+        "process_name": ["atera_agent.exe"]
+    },
+    "BeyondTrust (Bomgar)": {
+        "process_name": ["bomgar-scc.exe",
+                         "bomgar-rdp.exe"],
+        "domain": ["bomgarcloud.com"],
+        "digsig_publisher": ["Bomgar Corporation"]
+    },
+    "ConnectWise Control": {
+        "process_name": ["screenconnect.clientservice.exe",
+                        "screenconnect.windowsclient.exe"],
+        "domain": ["live.screenconnect.com"],
+        "digsig_publisher": ["ConnectWise, LLC"]
+    },
+    "Chrome Remote Desktop": {
+        "cmdline": ["cmdline:efmjfjelnicpmdcmfikempdhlmainjcb*"],
+        "domain": ["remotedesktop.google.com"]
+    },
+    "DameWare": { 
+        "process_name": ["dwrcs.exe"],
+        "internal_name": ["DWRCST"]
+    },
+    "Distant Desktop": {
+        "process_name": ["distant-desktop.exe"]
+    },
+    "DW Service": {
+        "process_name": ["dwagsvc.exe"],
+        "digsig_publisher": ["DWSNET OÜ"]
```

Comment:
@rc-zfink will that Ü character cause encoding issues in the searching for the platforms? 

@jholtmann - will special characters cause any problems with the API calls for any of the platforms?

Reply:
@rc-abodkins Carbon Black handles the characters fine. S1 was able to run the queries without visible errors, but I didn't a test environment to verify that the query is effective. I wasn't able to run any Surveyor queries for Defender - but I've also never tested the Defender integration so it's likely an issue with my setup.